### DISCUSSION TITLE
"node ." error

### DISCUSSION THREAD

**Discussion by northwl** [discordjs/discord.js#5913] [Q&A] [answered]
![unknown-2](https://user-images.githubusercontent.com/49620145/123211232-8916f500-d499-11eb-8ac9-d66beeca486a.png)

I need to know what this error means.
this happens when I run the "node ." at the terminal.

**Comment by kyranet** [comment, marked as answer]
You're creating a promise without a try/catch or a `.catch` handler. I assume this is on `client.login()`, so do `client.login().catch(console.error)` for example.
